### ISSUE TITLE
Support API versioning

### ISSUE THREAD

**Issue by vtourraine** [mixitconf/mixit#223]
I really appreciate that you offer an API to easily access the conference info. That being said, if I remember correctly, that’s the third year in a row that the API is changed significantly (different URLs, different data structure).

I’m probably over-thinking this (feel free to tag this as super-low-priority), but at this point, I think it would be great to add some kind of API versioning support. So if you decide to change the API next year, apps could handle the migration gracefully.

Practical suggestions:
- Specify HTTP header value (`Accept: application/vnd.myapi.v2 + json`)
- Specify custom HTTP header (`X-MyAPIVersionRequest-Header: 2`)
- Insert version number in API URLs (`http://api.mixitconf.org/v2/...`)

Many thanks!

**Comment by sdeleuze** [comment]
Good point, we will support that for next year, also the API should be much stable now. Thanks for your work on the Android app.

**Comment by clacote** [comment]
~Android~ iOS

**Comment by sdeleuze** [comment]
Oups :-)
